.github/workflows/build-and-push-image-prod.yml

```diff
@@ -0,0 +1,58 @@
+name: Continuous Delivery / Prod
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  set-test-release-version:
+    name: Set Test Release Version
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{steps.release-version.outputs.version}}
+    steps:
+      - id: release-version
+        run: echo "::set-output name=version::release-7"
+
+  build-and-push-image-test:
+    name: Build and push image test
```

Comment:
This should be `Production` as well, it's a small thing but potentially confusing.